loom/graphics/gfxGraphics.cpp

```diff
@@ -305,7 +356,7 @@ void Graphics::handleContextLoss()
 
 void Graphics::screenshot(const char *path)
 {
-    strcpy(pendingScreenshot, path);
+	strcpy(pendingScreenshot, path);
```

Comment:
Using a `utString` as the parameter instead of `const char*` should solve that issue, right?